addons/mail/static/src/models/discuss_sidebar_category_item/discuss_sidebar_category_item.js

```diff
@@ -34,7 +34,9 @@ function factory(dependencies) {
                 case 'group':
                     return `/web/image/mail.channel/${this.channel.id}/avatar_128?unique=${this.channel.avatarCacheKey}`;
                 case 'chat':
-                    return this.channel.correspondent.avatarUrl;
+                    if (this.channel.correspondent){
+                        return this.channel.correspondent.avatarUrl;
+                    }
```

Comment:
default if no match should be `return '/mail/static/src/img/smiley/avatar.jpg';`
It's already in livechat override, I would move it here below instead.

Reply:
Don't forget to remove the one in `im_livechat`, which is now a duplicate of the fallback here.